cmd/boots/main.go

```diff
@@ -96,14 +115,72 @@ func main() {
 			mainlog.Fatal(errors.Wrap(err, "retry syslog serve"))
 		}
 	}()
-
-	mainlog.Info("serving tftp")
-	go ServeTFTP()
+	g, ctx := errgroup.WithContext(ctx)
+	ipportHTTP, err := netaddr.ParseIPPort(c.HTTPAddr)
+	if err != nil {
+		mainlog.Fatal(err)
+	}
+	var nextServer net.IP
+	var httpServerFQDN string
+	if *rTFTP == "" && *rHTTP == "" {
+		mainlog.Info("serving ipxe binaries via tftp and http")
+		g.Go(func() error {
+			ipportTFTP, err := netaddr.ParseIPPort(c.TFTPAddr)
```

Comment:
I would love to have a way to disable TFTP :-)

Reply:
just TFTP, right? still serve iPXE binaries via HTTP. 

Is this because your hardware doesn't use TFTP to PXE boot?
I know this isn't disabling TFTP, but is setting `tftp-addr` to `127.0.0.1:8888` (for example) a satisfactory workaround?

Regardless, I'm looking into what it looks like to incorporate this option.

Reply:
> just TFTP, right? still serve iPXE binaries via HTTP.

Yes.

> Is this because your hardware doesn't use TFTP to PXE boot? I know this isn't disabling TFTP, but is setting the cli flag `-tftp-addr` to `127.0.0.1:8888` (for example) a satisfactory workaround?
> 
> Regardless, I'm looking into what it looks like to incorporate this option.

When its running in a container I guess it would.

Maybe just set it to an empty string? Or `--no-tftp`? Or `--no-tftp-addr`? I can even send a PR, we have to figure out which form of argument controls this.

Reply:
`no-tftp`, `tftp-disabled`, `disable-tftp`, etc all seem reasonable. Also, FYI: https://github.com/tinkerbell/boots-ipxe/pull/20

Reply:
@jacobweinstock, thx for https://github.com/tinkerbell/boots-ipxe/pull/20!

Adding the flag to the repository is the only thing missing? Do you plan on doing it? Or is it okay to wait for my PR (which I cannot say when I'll do it; maybe after a couple of weeks)?

Reply:
I'm waiting for 2 more PRs to land in https://github.com/tinkerbell/ipxedust/pulls and then I'll be updating this PR. I can add the flag then, assuming it's before you've started any work :)

Reply:
@rgl, PRs have landed. I've updated this draft PR. If you have a moment, please check it out and let me know what you think. thank you!